specification/conformance/conformance_rules/columns/commitmentdiscountquantity.json

```
@@ -0,0 +1,512 @@
+                        "ColumnName": "CommitmentDiscountId",
+                        "Value": null
+                    },
+                    {
+                        "CheckFunction": "CheckValue",
+                        "ColumnName": "ChargeClass",
+                        "Value": "Correction"
+                    }
+                ]
+            },
+            "Dependencies": []
+        }
+    },
+    "CommitmentDiscountQuantity-C-008-C": {
+        "Function": "Composite",
+        "Reference": "CommitmentDiscountQuantity",
+        "EntityType": "Column",
+        "Notes": "",
+        "CRVersionIntroduced": "1.2",
+        "Status": "Active",
+        "ApplicabilityCriteria": [],
+        "Type": "Static",
+        "ValidationCriteria": {
+            "MustSatisfy": "When CommitmentDiscountQuantity is not null, CommitmentDiscountQuantity adheres to the following additional requirements:",
+            "Keyword": "MUST",
+            "Requirement": {
+                "CheckFunction": "AND",
+                "Items": [
+                    {
+                        "CheckFunction": "CheckConformanceRule",
+                        "ConformanceRuleId": "CommitmentDiscountQuantity-C-009-M"
+                    },
+                    {
+                        "CheckFunction": "CheckConformanceRule",
+                        "ConformanceRuleId": "CommitmentDiscountQuantity-C-010-C"
+                    },
+                    {
+                        "CheckFunction": "CheckConformanceRule",
+                        "ConformanceRuleId": "CommitmentDiscountQuantity-C-013-C"
+                    }
+                ]
+            },
+            "Condition": {},
+            "Dependencies": [
+                "CommitmentDiscountQuantity-C-009-M",
+                "CommitmentDiscountQuantity-C-010-C",
+                "CommitmentDiscountQuantity-C-013-C"
+            ]
+        }
+    },
+    "CommitmentDiscountQuantity-C-009-M": {
+        "Function": "Validation",
+        "Reference": "CommitmentDiscountQuantity",
+        "EntityType": "Column",
+        "Notes": "",
+        "CRVersionIntroduced": "1.2",
+        "Status": "Active",
+        "ApplicabilityCriteria": [],
+        "Type": "Dynamic",
+        "ValidationCriteria": {
+            "MustSatisfy": "CommitmentDiscountQuantity MUST be a valid decimal value.",
+            "Keyword": "MUST",
+            "Requirement": {},
+            "Condition": {},
+            "Dependencies": []
+        }
+    },
+    "CommitmentDiscountQuantity-C-010-C": {
+        "Function": "Composite",
+        "Reference": "CommitmentDiscountQuantity",
+        "EntityType": "Column",
+        "Notes": "",
+        "CRVersionIntroduced": "1.2",
+        "Status": "Active",
+        "ApplicabilityCriteria": [],
+        "Type": "Static",
+        "ValidationCriteria": {
+            "MustSatisfy": "When ChargeCategory is “Purchase” adheres to the following additional requirements:",
```

We should also consider rephrasing this to reduce the nesting from 3 to 2 levels (see [this ](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/issues/1380#issuecomment-3383522798) comment in FR #1380:

```suggestion
            "MustSatisfy": "When ChargeCategory is \"Purchase\", CommitmentDiscountQuantity adheres to the following additional requirements:",
```